cmd/gcp-controller-manager/csr_approver.go

```diff
@@ -54,6 +55,16 @@ const (
 	authFlowLabelNone = "not_approved"
 )
 
+var (
+	// For the first startupErrorsThreshold after startupTime, label SAR errors
+	// differently.
+	// When kube-apiserver starts up (around the same time), it takes some time
+	// to setup RBAC rules for these SAR checks. This special labeling of
+	// errors lets us filter out the (expected) error noise at master startup.
```

Comment:
If you are intentionally disregarding leader election (which I think is ok), please add justification to godoc.

Reply:
Yeah, this is just to give kube-apiserver some time to initialize